### ISSUE TITLE
depth preprocessor runs on non-photo graphics, and should not

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#892]
It appears that depth preprocessor does not check the category of the graphic, so it is running on text, chart, and other categories. Since I believe it is only useful on photographs (@emmanuelwilson can ping if wrong!) it should be checking the output of the content-categoriser, just as other preprocessors like semseg and yolo do.

**Comment by emmanuelwilson** [comment]
Correct! It should check but I failed to add that functionality (sorry!). Should be a pretty straightforward fix
